protocol_check.py

```diff
@@ -19,7 +19,7 @@ def _get_parser():
     return parser
 
 
-def main(work_dir, bids_dir, sub, ses=None):
+def _main(work_dir, bids_dir, sub, ses=None, argv=None):
```

Comment:
Function should be public. Main goal of current structure is to allow pure Python usage.

Should also drop `argv`, which is just used to take in command-line args into argparser.